frontend/src/pages/org/workflows-list.ts

```diff
@@ -866,9 +857,9 @@ export class WorkflowsList extends BtrixElement {
             <button
               class="font-medium text-neutral-500 underline hover:no-underline"
               @click=${() => {
-                this.filterBy = {};
-                this.filterByCurrentUser = false;
-                this.filterByTags = undefined;
+                this.filterBy.setValue({});
+                this.filterByCurrentUser.setValue(false);
+                this.filterByTags.setValue(undefined);
```

Comment:
This should match "Clear All":
```suggestion
                this.clearFilters()
```